packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx

```diff
@@ -37,6 +37,9 @@ const SendButton = styled.button({
   ':hover': {
     filter: 'brightness(0.8)',
   },
+  ':enabled': {
+    background: 'var(--color-surprise)',
```

Comment:
@marckong I think the text when the button is enabled is not super clear, perhaps we should also change it's color. Here's a suggestion:
```suggestion
    background: 'var(--color-surprise)',
    color: 'var(--color-font-surprise)';
```

Before:
![image](https://user-images.githubusercontent.com/11976836/189599551-e137ed1a-677b-483b-8fb5-511f35259980.png)

After:
![image](https://user-images.githubusercontent.com/11976836/189599476-5b9b4b3f-7816-41e5-a76e-4df378b1a25a.png)